orchestra/bots/staffbot.py

```diff
@@ -26,28 +32,75 @@ class StaffBot(BaseBot):
         (r'staff (?P<task_id>[0-9]+)', 'staff'),
         (r'restaff (?P<task_id>[0-9]+) (?P<username>[\w.@+-]+)', 'restaff'),
     )
+    task_does_not_exist_error = 'Task {} does not exist'
+    task_assignment_error = 'Task {} got an error: "{}"'
+    worker_does_not_exist = 'Worker with username {} does not exist'
+    task_assignment_does_not_exist_error = (
+        'TaskAssignment associated with user {} and task {} does not exist.')
 
     def __init__(self, **kwargs):
         default_config = getattr(settings, 'STAFFBOT_CONFIG', {})
         default_config.update(kwargs)
         token = settings.SLACK_STAFFBOT_TOKEN
         super().__init__(token, **kwargs)
 
-    def staff(self, task_id):
+    def staff(self, task_id, **kwargs):
```

Comment:
why not `def staff(self, task_id, request_cause=StaffingRequest.RequestCause.USER.value):`